Lib9c.Policy/AccessControlService/IAccessControlService.cs

```diff
@@ -5,5 +5,7 @@ namespace Nekoyume.Blockchain
     public interface IAccessControlService
     {
         public bool IsAccessDenied(Address address);
```

Comment:
Is it going to be deprecated?

Reply:
Technically, yes. But, I wanted to preserve it just incase we might need to use it later 😅 Should we just remove it?

Reply:
No... I agree with you, but we should write a deprecation comment for `IsAccessDenied`